src/server/api/routers/webhook.ts

```diff
@@ -3,54 +3,97 @@ import { z } from "zod";
 import { env } from "~/env.mjs";
 import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
 
+interface GitHubAPIError {
+  status: number;
+}
+
 export const webhookRouter = createTRPCRouter({
-  createWebhook: protectedProcedure.input(z.object({ accessToken: z.string()})).mutation(async ({ input }) => {
+  // accessToken is optional because there can be instances where the user's session is remembered (automatically is redirected to hompage) 
+  // which makes the accessToken undefined since accessToken is grabbed when user is loggin in (authenticating).
```

Comment:
logging

Reply:
logging in what sense? i've ran some logs and the session's accessToken is a short-lived token so updates would be occuring.

Reply:
oh i just realized i wrote loggin 😅😂